mscore/importmxmlpass2.h

```diff
@@ -220,11 +235,12 @@ class MusicXMLParserPass2 {
 
       // part specific data interface functions
       void addSpanner(const MusicXmlSpannerDesc& desc);
-      SLine* getSpanner(const MusicXmlSpannerDesc& desc);
+      MusicXmlExtendedSpannerDesc& getSpanner(const MusicXmlSpannerDesc& desc);
       void clearSpanner(const MusicXmlSpannerDesc& desc);
 
 private:
       void initPartState(const QString& partId);
+      SpannerSet FindIncompleteSpannersAtPartEnd();
```

Comment:
Could you please rename this function to start from a small `f` letter? Names started with capital letter are usually reserved for names of types and enumerators within MuseScore code, and MusicXML import code seems to follow this rule as well.

Reply:
Sure, thanks for catching my mistake.